payments/utils.py

```diff
@@ -156,6 +157,21 @@ def calculate_product_percentage_price(base_price, percentage):
     return base_price * (percentage / 100)
 
 
+def calculate_organization_price(price, organization_type: OrganizationType) -> Decimal:
+    if organization_type == OrganizationType.COMPANY:
+        return price * 2
+    elif organization_type == OrganizationType.NON_BILLABLE:
+        return Decimal("0.00")
+    else:
+        return price
```

Comment:
Would it make more sense to do the
```python 
return rounded_price(price)
```

instead of later having to call it? Since we already know it's gonna be a price
```python
organization_price = calculate_organization_price(
    price_value, organization_type
)

self.price = rounded_decimal(organization_price)
```

Reply:
That function is used in two places where another one (in OrderLine) does not call rounding, so that's why it doesn't do the rounding.

Reply:
Let's skip the later call and add it here instead.

Reply:
done